QUARKUS-5867.MD

```diff
@@ -0,0 +1,64 @@
+# QUARKUS-5867 - Micrometer to OpenTelemetry Bridge
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5867
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/43621
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/43831
+
+Quarkus 3.19 introduced a new Quarkus Micrometer to OpenTelemetry Bridge extension with a tech preview status.
+The extension integrates the `io.opentelemetry.instrumentation:opentelemetry-micrometer-1.5` library
+with the Quarkus OpenTelemetry and Quarkus Micrometer extensions, thus allowing to support both Micrometer and OpenTelemetry
+in one Quarkus application, without the need to export the metrics into two collectors.
+The Micrometer semantic convention is not compatible with the OpenTelemetry semantic convention; therefore,
+Micrometer metrics are mapped to the OpenTelemetry metrics.
+
+## Scope of the testing
+
+New test coverage should verify that:
+
+- all telemetry data (logs, tracing, metrics) created with both Micrometer and OpenTelemetry processed by the same OpenTelemetry SDK instance, and will use its configuration and exporters
+- Micrometer-specific configuration is possible even when using the bridge extension, for example, the following must work:
```

Comment:
Please make sure that Micrometer automatically generated metrics are also propagated (e.g. pick few and check them)

https://quarkus.io/guides/telemetry-micrometer#automatically-generated-metrics